cmd/kubeadm/app/cmd/util/join.go

```diff
@@ -32,7 +32,7 @@ import (
 var joinCommandTemplate = template.Must(template.New("join").Parse(`` +
 	`kubeadm join {{.ControlPlaneHostPort}} --token {{.Token}} \
     {{range $h := .CAPubKeyPins}}--discovery-token-ca-cert-hash {{$h}} {{end}}{{if .ControlPlane}}\
-    --experimental-control-plane {{if .CertificateKey}}--certificate-key {{.CertificateKey}}{{end}}{{end}}`,
+    --control-plane {{if .CertificateKey}}--certificate-key {{.CertificateKey}}{{end}}{{end}}`,
```

Comment:
The same.

Remove `ExperimentalControlPlane` as well and use `UploadCerts` instead.

Reply:
@dixudx 
I looked at the code again and found this
```
// NewCmdUpgradeControlPlane returns the cobra.Command for upgrading the controlplane instance on this node
// TODO: to remove when 1.16 is released
func NewCmdUpgradeControlPlane() *cobra.Command {
	nodeOptions := newNodeOptions()
	nodeRunner := workflow.NewRunner()

	cmd := &cobra.Command{
		Use:        "experimental-control-plane",
		Short:      "Upgrade the control plane instance deployed on this node. IMPORTANT. This command should be executed after executing `kubeadm upgrade apply` on another control plane instance",
		Deprecated: "this command is deprecated. Use \"kubeadm upgrade node\" instead",
		Run: func(cmd *cobra.Command, args []string) {
			err := nodeRunner.Run(args)
			kubeadmutil.CheckErr(err)
		},
	}
...
```
Is it safe to remove this code now ?
Do we need to wait until 1.16 is released ?
/cc @neolit123 @rosti

Reply:
again we missed changing this line in 1.15, but `--experimental-control-plane` still works there so it's fine.

> I looked at the code again and found this

i would leave command changes to a separate PR, as this one is for STDOUT changes.
in a separate PR we can remove all commands that have `experimental` and should be removed for 1.16 (master).

e.g.
```
init phase upload-certs experimental-upload-certs
upgrade experimental-control-plane
join experimental-control-plane
...
```
just grep for experimental under cmd/kubeadm

Reply:
yep, it is safe to remove in 1.16.